README.md

```diff
@@ -0,0 +1,42 @@
+YaST - CaaSP integration
+========================
+
+[![Travis Build](https://travis-ci.org/yast/yast-caasp.svg?branch=master)](https://travis-ci.org/yast/yast-caasp)
+[![Coverage Status](https://img.shields.io/coveralls/yast/yast-caasp.svg)](https://coveralls.io/r/yast/yast-caasp?branch=master)
+[![Jenkins Build](http://img.shields.io/jenkins/s/https/ci.opensuse.org/yast-caasp-master.svg)](https://ci.opensuse.org/view/Yast/job/yast-caasp-master/)
+[![Code Climate](https://codeclimate.com/github/yast/yast-caasp/badges/gpa.svg)](https://codeclimate.com/github/yast/yast-caasp)
+
+This repository contains code which is specific to the Containers as a Service Platform
+(CaaSP) product.
+
+
+Provided Functionality
+======================
+
+See [the generated yardoc documentation](http://www.rubydoc.info/github/yast/yast-caasp) at rubydoc.info.
+
+
+Getting the Sources
+===================
+
+To get the source code, clone the GitHub repository:
+
+    $ git clone https://github.com/yast/yast-caasp.git
+
+If you want to contribute into the project you can
+[fork](https://help.github.com/articles/fork-a-repo/) the repository and clone your fork.
```

Comment:
The `CONTRIBUTING.md` file is added in the other https://github.com/yast/yast-caasp/pull/1. About referencing it here, I just followed the same approach we have in the README of other yast2 projects (like yast2 or yast2-installation).